elmslie-core/src/main/java/vivid/money/elmslie/core/store/linking/LinkingExt.kt

```diff
@@ -0,0 +1,69 @@
+package vivid.money.elmslie.core.store.linking
+
+import vivid.money.elmslie.core.store.Store
+
+fun <
+    ParentEvent : Any,
+    ParentState : Any,
+    ParentEffect : Any,
+    ChildEvent : Any,
+    ChildState : Any,
+    ChildEffect : Any,
+> Store<ParentEvent, ParentState, ParentEffect>.linkTo(
+    childStore: Store<ChildEvent, ChildEffect, ChildState>,
+    effectMapper: (ChildEffect) -> ParentEvent?,
+    stateMapper: (ChildState) -> ParentEvent?,
+) = apply {
+    launch {
+        childStore.states().collect { stateMapper.invoke(it)?.let(this@linkTo::accept) }
+        childStore.effects().collect { effectMapper.invoke(it)?.let(this@linkTo::accept) }
+    }
+}
+
+fun <
+    ParentEvent : Any,
+    ParentState : Any,
+    ParentEffect : Any,
+    ChildEvent : Any,
+    ChildState : Any,
+    ChildEffect : Any,
+> Store<ParentEvent, ParentState, ParentEffect>.linkTo(
+    childStore: Store<ChildEvent, ChildEffect, ChildState>,
+    effectMapper: (ChildState, ChildEffect) -> ParentEvent? = { _, _ -> null },
+    stateMapper: (ChildState, ChildState) -> ParentEvent? = { _, _ -> null },
+) = apply {
```

Comment:
Maybe leave only this version of the method?

Reply:
one for fragments, another for activities, we can replace it with one method, but it will not be so easy to use

Reply:
It's placed on core module, and there are none of fragment/activites etc)